pythonforandroid/recipes/libzmq/__init__.py

```diff
@@ -31,14 +31,14 @@ def build_arch(self, arch):
             bash = sh.Command('sh')
             shprint(
                 bash, './configure',
-                '--host=arm-linux-androideabi',
+                '--host=' + arch.command_prefix,
                 '--without-documentation',
                 '--prefix={}'.format(prefix),
                 '--with-libsodium=no',
                 _env=env)
             shprint(sh.make, _env=env)
             shprint(sh.make, 'install', _env=env)
-            shutil.copyfile('.libs/libzmq.so', join(
+            shutil.copyfile('src/.libs/libzmq.so', join(
```

Comment:
When I tried locally it seemed like the file was still in `.libs/libzmq.so` hence without the `src/` prefix you added

Reply:
it's weird because in the last version of libzmq from (github), they generate output in this path, in the old zip not

Reply:
Weird indeed, let's see if Travis builds green, but I have my doubts because I'm running in an environment pretty close to it (Ubuntu 18.04)

Reply:
Good news, seems like it built OK in Travis :tada: 
I will merge